### PR TITLE
Update termux.md

### PR DESCRIPTION
**Pull request by mrfoxie**
Direct installation of `code-server` via `yarn` method in termux android version

### CODE REVIEW

File: docs/termux.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # Termux
 
 - [Install](#install)
+- [Direct Installation - Yarn Installation](yarn-installation)
```
Comment (jsjoeio) [inline docs/termux.md:6]:
```suggestion
- [Direct Installation via yarn](#yarn-installation)
```

File: docs/termux.md [hunk 2]
```diff
@@ -65,6 +66,50 @@ curl -fsSL https://code-server.dev/install.sh | sh
 > Consider using a new user instead of root, read [here](https://www.howtogeek.com/124950/htg-explains-why-you-shouldnt-log-into-your-linux-system-as-root/) why using root is not recommended.\
 > Learn how to add a user [here](#create-a-new-user).
 
+## Yarn Installation
+
+1. Get [Termux](https://f-droid.org/en/packages/com.termux/) from **F-Droid**.
```
Comment (jsjoeio) [inline docs/termux.md:71]:
```suggestion
1. Install [Termux](https://f-droid.org/en/packages/com.termux/) from **F-Droid**.
```

File: docs/termux.md [hunk 3]
```diff
@@ -65,6 +66,50 @@ curl -fsSL https://code-server.dev/install.sh | sh
...
+
+2. We will now change using the following command.
```
Comment (jsjoeio) [inline docs/termux.md:73]:
```suggestion
2. Run the following command:
```

File: docs/termux.md [hunk 4]
```diff
@@ -65,6 +66,50 @@ curl -fsSL https://code-server.dev/install.sh | sh
...
+```
+Now select `Main Repository` then change repo to `Mirrors by Grimler Hosted on grimler.se`.
+
+3. After successfully updating of repository update and upgrade all the packages by the following command
+```sh
+pkg update
+pkg upgrade -y
+```
+
+4. Now let's install requirement dependancy.
```
Comment (jsjoeio) [inline docs/termux.md:85]:
```suggestion
4. Now let's install required dependencies:
```

File: docs/termux.md [hunk 5]
```diff
@@ -65,6 +66,50 @@ curl -fsSL https://code-server.dev/install.sh | sh
...
+npm config set python python3
+node -v
+```
+you will get node version `v16.15.0`
+
+5. Now install code-server
+```sh
+yarn global add code-server
+```
+6. Congratulation code-server is installed on your device using the following command.
```
Comment (jsjoeio) [inline docs/termux.md:103]:
```suggestion
6. Congratulations! code-serve can now be installed on your device using the following command:
```

File: docs/termux.md [hunk 6]
```diff
@@ -65,6 +66,50 @@ curl -fsSL https://code-server.dev/install.sh | sh
...
+```sh
+code-server --auth none
+```
+7. If already installed then use the following command for upgradation.
```
Comment (jsjoeio) [inline docs/termux.md:107]:
```suggestion
7. To upgrade, run this command:
```

**Review by jsjoeio — CHANGES_REQUESTED** [review]
Thanks for fixing this! Added some suggestions then we can merge

**Comment by mrfoxie** [comment]
> Thanks for fixing this! Added some suggestions then we can merge

😅😅 sorry for silly mistake

**Comment by jsjoeio** [comment]
Can you run `yarn fmt` locally?

**Review by jsjoeio — APPROVED** [review]

**Comment by mrfoxie** [comment]
there is an another way also of installing code-server on Android but it requires another way of installation rather then termux it will take some time but i will be surely adding it into docs sections

**Pull request merged**